shared/constants/multichain/assets.ts

```diff
@@ -0,0 +1,5 @@
+import { MultichainNetworks } from './networks';
+
+export const MULTICHAIN_NATIVE_CURRENCY_TO_CAIP19 = {
```

Comment:
We will change this to an actual `enum` with the account overview PR: https://github.com/MetaMask/metamask-extension/pull/25200